README.md

```diff
@@ -406,9 +406,11 @@ This repo can then be utilized to apply changes to the exercises via interactive
 linearize 1.x
 Usage: linearize [options] masterRepo linearRepo
 
-  masterRepo          base folder holding master course repository
-  linearRepo          base folder for linearized version repo
-  -mjvm, --multi-jvm  generate multi-jvm build file
+  masterRepo               base folder holding master course repository
+  linearRepo               base folder for linearized version repo
+  -mjvm, --multi-jvm       generate multi-jvm build file
+  -f, --force-delete-existing-destination-folder
+                           generate multi-jvm build file
```

Comment:
Is generate multi-jvm build file repeated here? No description for -f?

Also --force-delete-existing-destination-folder is a really long flag. I can't imagine anyone ever typing that. I would go with --force-delete or even just --force.